networking_romana/driver/ipam_romana.py

```diff
@@ -170,27 +160,15 @@ def __init__(self, host_id, tenant_id, segment_id):
 
 
 class RomanaAddressRequestFactory(ipam_req.AddressRequestFactory):
-    """Builds request using ip information.
-
-    Additional parameters(port and context) are not used in default
-    implementation, but planned to be used in sub-classes
-    provided by specific ipam driver.
-    """
+    """Builds address request using ip information."""
 
     _db_url = None
     _db_conn_dict = None
 
     @classmethod
     def get_request(cls, context, port, ip_dict):
-        """Get a prepared Address Request.
-
-        :param context: context (not used here, but can be used in sub-classes)
```

Comment:
Descriptions of parameters don't need to be removed, right? The "not used here, blah, blah..." comment might go, but the parameter description itself is not bad.